glean-core/ios/Glean.xcodeproj/project.pbxproj

```diff
@@ -735,6 +735,7 @@
 				);
 				INFOPLIST_FILE = Glean/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
What does this do? Does it force a minimum of iOS 11?

Reply:
No it's just a target deployment, like the target version of Android doesn't have to be the minimum supported SDK version.